### PR TITLE
Refactor photometry detection threshold nsigma to be configurable

### PR DESCRIPTION
**Pull request by kmshin1397**
There are multiple places throughout the codebase where some n-sigma value/ signal-to-noise ratio is hard-coded and used to determine whether a photometry point is a detection or not. This PR ties all that to a single value in the configs for consistency/configurability. 

Would particularly appreciate careful review of the inline comments I modified to make sure that they make sense in the scientific sense :)

### CODE REVIEW

File: skyportal/schema.py [hunk 1]
```diff
@@ -664,9 +668,9 @@ def parse_mag(self, data, **kwargs):
             flux = 10 ** (-0.4 * (data['mag'] - PHOT_ZP))
             fluxerr = data['magerr'] / (2.5 / np.log(10)) * flux
         else:
-            fivesigflux = 10 ** (-0.4 * (data['limiting_mag'] - PHOT_ZP))
+            nsigflux = 10 ** (-0.4 * (data['limiting_mag'] - PHOT_ZP))
             flux = None
-            fluxerr = fivesigflux / 5
+            fluxerr = nsigflux / data['limiting_mag']
```
Comment (dannygoldstein) [inline skyportal/schema.py:673]:
```suggestion
            fluxerr = nsigflux / PHOT_DETECTION_THRESHOLD
```

**Review by dannygoldstein — CHANGES_REQUESTED** [review]
Heroic work. Thanks.

**Review by dmitryduev — APPROVED** [review]
Thanks @kmshin1397! Feel free to merge once @dannygoldstein's suggestions are in.

**Comment by kmshin1397** [comment]
@dannygoldstein Sorry, forgot to come back to this until now. Your suggestion is now implemented

**Comment by dannygoldstein** [comment]
@kmshin1397 merge conflicts

**Comment by kmshin1397** [comment]
@dannygoldstein fixed

**Pull request merged**